packages/components/src/Tooltip/Tooltip.tsx

```diff
@@ -21,9 +21,15 @@ interface TooltipProps {
    * Tooltip text
    */
   readonly message: string;
+
+  readonly setTabIndex?: boolean;
```

Comment:
Forcibly having a tabIndex of zero on the top level of the Chip was interfering with tab and keyboard based controls for Chips so we had to make setting the tabIndex configurable.